suite/update.sh

```diff
@@ -26,7 +29,10 @@ do
     scope=$(dirname $dep)
     case $scope in
         "@ceramicnetwork")
-            filtered_deps="$filtered_deps $dep@$target"
+            target_version=$(npm view $dep@$target version)
+            if [[ -n "$target_version" ]] && semver -r ">$current_version" $target_version &> /dev/null; then
```

Comment:
Will semver be installed? If not maybe we can install it as a dev dependency in packages.json?

Reply:
yeah, added this as devDependecy just to be sure